snuba/admin/static/clickhouse_migrations/index.tsx

```diff
@@ -17,6 +17,15 @@ function ClickhouseMigrations(props: { api: Client }) {
   const [migrationId, setMigrationId] = useState<string | null>(null);
   const [SQLText, setSQLText] = useState<string | null>(null);
 
+  const [header, setHeader] = useState<string | null>(null);
+  const [show_action, setShowAction] = useState<boolean | null>(false);
+
+  const dry_run_header = "Raw SQL for running a migration (forwards) or reversing \n (backwards). Good to do before executing a migration for real."
+  const real_run_header = "Run log output"
+
+  const [forwards_dry_run, setFowardsDryRun] = useState<string | null>(null);
+  const [backwards_dry_run, setBackwardsDryRun] = useState<string | null>(null);
```

Comment:
also might be nice to show the buttons as selected since you have this state now.

Reply:
I fixed it to give the buttons a red text when selected.